src/core/command/composite.js

```diff
@@ -0,0 +1,33 @@
+import {Command} from './command';
+import {noop} from '../utility';
+
+export class Composite {
+	static function (functions, reducer = noop, memo = null) {
```

Comment:
maybe func? function is so system :)

Reply:
do we need reducer and memo

Reply:
I think, in general case, it might be useful because potentially there can be a case when user needs to handle returned value somehow